src/dos/dos_keyboard_layout.cpp

```diff
@@ -38,6 +38,16 @@ using sv = std::string_view;
 #include "setup.h"
 #include "string_utils.h"
 
+extern void DOS_UpdateCurrentProgramName();
+
+static void notify_new_code_page(const bool reason_keyboard_layout = false)
```

Comment:
I have no idea what `reason_keyboard_layout` means.

Reply:
Renamed variable to `keyboard_layout_changed`, hopefully it is more clear now.